tags/java.md

```diff
@@ -1 +1,8 @@
-https://prismlauncher.org/wiki/getting-started/installing-java/
+---
+embed:
+  title: Java Instructions 
+  color: orange
+---
+
+For security reasons, Prism Launcher does not bundle in Java with itself. The instructions to setup and install Java can be found [here](https://prismlauncher.org/wiki/getting-started/installing-java/).
```

Comment:
I think this isn't because of "security reasons" but more because it isn't implemented yet?